app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
The `people` dictionary is a class variable, which means it is shared across all instances of the `Person` class. This can lead to unexpected behavior, such as overwriting data if multiple `Person` objects with the same name are created. Consider using an instance variable instead.